### ISSUE TITLE
Epic: Accessibility Phase 1 - Panels

### ISSUE THREAD

**Issue by tskarhed** [grafana/grafana#36776]
We should check if there are any improvements we can do to the panel plugins to make it more accessible too.

## Panel chrome
- [ ] Panels should be keyboard navigable ![avatar](https://github.com/kaydelaney.png?size=24) #37613
  - [ ] Panel menus should be navigable with keyboard controls (use aria-expanded here)
- [ ] Make information and warning triangle accessible. Could be solved if Tooltips are made accessible.
 
## Panel Plugins
- [ ] Table
  - [ ] Make sure all rows are readable
  - [ ] Remove unnecessary sections. Causes confusion with VoiceOver
- [ ] Time series panel
  - [ ] See what low hanging fruit we can get here. The sonification project could probably fit in
- [ ] Gauge, bar chart, stat
  - [ ] Make the values readable in a decent way
- [ ] Bar chart: Make key-values more easily read
- [ ] Gauge: Make image invisible for screen readers
- [ ] Pie chart: Always read percentages for screen readers
- [ ] Row: Add section for row
  - [ ] labels for icons
- [ ] Alert list:
  - [ ] Increase color contrast
  - [ ] Make keyboard expandable
  - [ ] aria-hidden on some icons
- [ ] Dashboard list: Increase color contrast

We should also figure out a way to communicate panel shortcuts.

**Comment by tskarhed** [comment]
Closing this, we will revisit. This is mainly for DataViz to handle.
